tools/infrasys/src/shared.rs

```diff
@@ -66,10 +68,9 @@ pub async fn get_stack_outputs(
         );
         thread::sleep(time::Duration::from_secs(20));
         stack_outputs = cfn_client
-            .describe_stacks(DescribeStacksInput {
-                stack_name: Some(stack_name.to_string()),
-                ..Default::default()
-            })
+            .describe_stacks()
+            .stack_name(stack_name.to_string())
```

Comment:
This and various other places shouldn't need `to_string()` as the input is `Into<String>`.